.github/workflows/zxc-build-library.yaml

```diff
@@ -107,7 +107,7 @@ jobs:
       #     run: tree /F build
 
       - name: Start the local node
-        run: sudo npx @hashgraph/hedera-local start -d --network local --network-tag=0.51.0-alpha.3
+        run: sudo npx @hashgraph/hedera-local start -d --network local --network-tag=0.52.0-alpha.4
```

Comment:
`0.52.0-alpha.4` should we try with `alpha.6` before merging as you mentioned in the PR comments?